src/interfaces/protocols/handlers/protocols-configure.ts

```diff
@@ -56,7 +58,8 @@ export class ProtocolsConfigureHandler implements MethodHandler {
       const { author } = protocolsConfigure;
       const indexes = {
         author,
-        ... message.descriptor
+        ... message.descriptor,
```

Comment:
NIT: we probably could/should move this to be before `author` just in case it's inside `message.descriptor` so that it's not overridden by the `...`